### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by tapaswenipathak**
Ref https://github.com/QubesOS/qubes-issues/issues/4260.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+qubes-gui-daemon
+================
+
+gui-daemon is Qubes GUI Virtualisation agent/daemon, present on Dom0 side.
+
+gui-daemon has default daemon setting. Starting the GUI will ask you whether to
+start the daemon and you can then accept. The GUI domain would be the same as
```
Comment (marmarek) [inline README.md]: Can you explain what you mean by "the GUI will ask you whether to start the daemon and you can then accept"? There is no interactive prompt, just command like arguments.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+admin domain in all but few scenarios. With just few `#ifdefs` the existing
+GUI daemon should compile and run on most OS from Linux/Xorg, Windows and MAcs.
```
Comment (marmarek) [inline README.md]: If you mean running gui-daemon on non-Xorg system, this is not true. Much of the code is very much Xorg specific. It is definitely possible to add support for other OSes, but unfortunately it isn't "just few `#ifdefs`".

File: README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+The config is defined by guid.conf.
+
+![](./assets/qubes-components.png)
```
Comment (marmarek) [inline README.md]: It may be also good to link to https://www.qubes-os.org/doc/gui/

File: README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+## Building qubes-gui-daemon
+
+make `gui-daemon`
```
Comment (marmarek) [inline README.md]: This is true, but an important detail is missing: it should be run from qubes-builder, not this repository directly. https://www.qubes-os.org/doc/qubes-builder/

Alternatively `make all BACKEND_VMM=xen` here. But will require installing dependencies manually.

**Comment by tapaswenipathak** [comment]
@marmarek how do you suggest adding me the qubes-components image? What do you have in mind?

File: README.md (hunk 2 above)
Comment (tapaswenipathak) [reply README.md]: From my observation you can have the enablement if you use libvirt.

**Comment by marmarek** [comment]
> @marmarek how do you suggest adding me the qubes-components image? What do you have in mind?

Linking to https://www.qubes-os.org/attachment/wiki/posts/qubes-components.png

**Comment by tapaswenipathak** [comment]
@marmarek looks ok?

**Comment by tapaswenipathak** [comment]
@marmarek looks ok?

File: README.md (hunk 1 above)
Comment (marmarek) [reply README.md]: This comment still apply

File: README.md (hunk 2 above)
Comment (marmarek) [reply README.md]: It is possible to add support for other operating systems. But the code as it is right now supports only Linux/Xorg.

**Comment by marmarek** [comment]
While Whonix do use mediawiki format, we try to not add other formats than markdown (`.md`) and in some places reStructuredText (`.rst`). So, please convert to markdown.

File: README.md (hunk 1 above)
Comment (tapaswenipathak) [reply README.md]: Could you write the statement properly?

**Comment by tapaswenipathak** [comment]
@marmarek Just curious have you implemented any `.mediaiwiki` file formatting before? Just as an FYI there is no use of having `.mediawiki` formatting if you use `.md` formatting, sample commit cb9c52d. Why not have only `.md` then? What is the use of changing the file formatting as per `.mediawiki` rules?

**Pull request closed without merging**